src/Components/LogTable.js

```diff
@@ -144,9 +144,12 @@ export class LogTable extends Component {
 		}
 
 		const levelHeader = (
-			<>{t('logreader', 'Level')}
+			<>
+			<button>
+			{t('logreader', 'Level')}
 				<span className={style['log-settings-toggle'] + ' icon-more'}>
 				</span>
+			</button>
 			</>
```

Comment:
```suggestion
			<button>
				{t('logreader', 'Level')}
				<span className={style['log-settings-toggle'] + ' icon-more'}>
				</span>
			</button>
```

Don't need a fragment there anymore, as everything is wrapped with button